lib/mayaUsd/ufe/UsdMaterial.cpp

```diff
@@ -98,14 +119,20 @@ std::vector<Ufe::SceneItem::Ptr> UsdMaterial::getMaterials() const
         if (material) {
             for (const auto& output : material.GetSurfaceOutputs()) {
```

Comment:
Same thing here. You already have the material an do not need to traverse any surface output.